orttraining/orttraining/python/training/ortmodule/_gradient_accumulation_manager.py

```diff
@@ -0,0 +1,80 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+from . import _utils
+from onnxruntime.capi import _pybind_state as C
+
+class GradientAccumulationManager(object):
+    """
+    GradientAccumulationManager is resposible for handling Gradient accumulation optimization
+    during training
+    """
```

Comment:
```suggestion
    """Handles Gradient accumulation optimization during training
    
    Args:
        enabled (bool, optional): Enables gradient accumulation when True. Default is False
    """
```

ORTModule uses Google Code style (https://google.github.io/styleguide/pyguide.html). Basically it has a short first line, followed by a empty line and optional "Args:", "Returns:", "Raises:" depending on the case

Reply:
One information that would add in the class docstring is (after the new line and before Args: Can this feature be enabled/disabled during training or this is a one-time enabling/disabling and this is it?

Reply:
got it. I have added the docstring stating that it can only be turned on once before training, and can't be turned off, and added a TODO to enable switching it on/off  during training. As it is not a common scenario, we can revisit it in the future.